README.md

```diff
@@ -1 +1,185 @@
-# BLOG-app.
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Blog App <a name="about-project"></a>
+
+A **Ruby blog app** The web application is designed to enable users to create, edit, and publish blog posts. It is developed using the Ruby on Rails framework, a popular choice known for its speed and user-friendly features in web application development.
+
+Additionally, the application functions as a standard blog website, offering a comprehensive and functional platform. Users can browse a collection of articles and interact with them by leaving comments and expressing their appreciation through features like liking posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+    <li><a href="https://rubocop.org/">Rubocop</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The key features of this project includes.
+
+- **Display Users**
+- **Display Posts**
+- **Display Likes on Posts**
+- **Display Comments on Posts**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+The Live Demo of this Project will be available soo.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<ul>
+    <li><p>A code editor (Visual Studio Code is recommended)</p></li>
+</ul>
+
+<ul>
+    <li><p>Git and a GitHub account</p></li>
+</ul>
+
+### Setup
+
+Clone this repository to your desired folder by using this 👇️ command :
+
+```
+git clone https://github.com/firstpj/blog-app.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd blog-app
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+  rails s
+```
+
+### Run tests
+
+- Tests are not available yet, please stay tuned for it.
```

Comment:
- [ ] This Project milestone is related with test cases for your blog app So kindly mention  here how to run the test cases locally on machine.